sumologic/sumologic_client.go

```diff
@@ -42,7 +42,7 @@ func createNewRequest(method, url string, body io.Reader, accessID string, acces
 		return nil, err
 	}
 	req.Header.Add("Content-Type", "application/json")
-	req.Header.Add("User-Agent", "SumoLogicTerraformProvider/2.0.2")
+	req.Header.Add("User-Agent", "SumoLogicTerraformProvider/2.1.3")
```

Comment:
We should find a better way to do this. This should be updated for each release.

Reply:
yup, I've pretty much forgotten every release.